src/custom/state/price/reducer.ts

```diff
@@ -23,9 +24,11 @@ export interface PriceInformation {
 }
 
 export interface QuoteInformationObject extends FeeQuoteParams {
-  fee: FeeInformation
-  price: PriceInformation
+  fee?: FeeInformation
+  price?: PriceInformation
+  error?: ApiErrorCodes
```

Comment:
Are you sure we want to create 1<>1 relation with this error to `ApiErrorCodes` backend codes?

I would think that we just want a new enum with the errors we handle? Maybe I should wait for your next PR and see  how you use this?

Reply:
thought about it but don't see any immediate danger doing it this way

Reply:
noted in #751